components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java

```diff
@@ -490,6 +493,10 @@ public RefreshTokenValidationDataDO validateRefreshToken(String consumerKey, Str
                     "TOKEN_SCOPE, TOKEN_STATE, TIME_CREATED FROM " + accessTokenStoreTable +
                     " WHERE CONSUMER_KEY = ? AND REFRESH_TOKEN = ? ORDER BY TIME_CREATED DESC LIMIT 1";
 
+            db2Query = "SELECT ACCESS_TOKEN, AUTHZ_USER, " +

```

Comment:
We have moved all these queries to SQLQueries.java. Can you define this query at SQLQueries.java and refer it from here. Please refer PR at https://github.com/wso2/carbon-identity/pull/610/files